SPECS/edk2/edk2.spec

```diff
@@ -108,6 +108,7 @@ Patch0014: 0014-SecurityPkg-add-TIS-sanity-check-tpm2.patch
 Patch0015: 0015-SecurityPkg-add-TIS-sanity-check-tpm12.patch
 Patch0016: 0016-OvmfPkg-Clarify-invariants-for-NestedInterruptTplLib.patch
 Patch0017: 0017-OvmfPkg-Relax-assertion-that-interrupts-do-not-occur.patch
+Patch0018: CVE-2024-1298.patch
```

Comment:
LGTM :)
Good call on using 0018 instead of 1005 because we only auto apply patches till <= 999

Reply:
yes